AsyncDisplayKit/TextKit/ASTextKitContext.h

```diff
@@ -30,11 +32,20 @@
                          constrainedSize:(CGSize)constrainedSize
               layoutManagerCreationBlock:(NSLayoutManager * (^)(void))layoutCreationBlock
                    layoutManagerDelegate:(id<NSLayoutManagerDelegate>)layoutManagerDelegate
-                textStorageCreationBlock:(NSTextStorage * (^)(NSAttributedString *attributedString))textStorageCreationBlock;
+                textStorageCreationBlock:(ASTextKitContextTextStorageCreationBlock)textStorageCreationBlock;
 
+/**
+ Set the constrained size for the text context.
+ */
 @property (nonatomic, assign, readwrite) CGSize constrainedSize;
 
 /**
+ Resets the text storage to the original value in case it was truncated before. This method is called within
```

Comment:
Could you clarify in the comment what "the original value" means?  Is that the original, un-truncated string?  It seems like there might be a few possible things that could be reset (e.g. like clearing the calculated size).